packages/block-editor/src/components/inserter/block-patterns-tab.js

```diff
@@ -144,6 +149,12 @@ export function usePatternsCategories( rootClientId, sourceFilter = 'all' ) {
 				label: _x( 'Uncategorized' ),
 			} );
 		}
+		if ( filteredPatterns.some( ( pattern ) => pattern.id ) ) {
+			categories.unshift( {
+				name: myPatternsCategory.name,
+				label: myPatternsCategory.label,
+			} );
```

Comment:
```suggestion
			categories.unshift( myPatternsCategory );
```

Can we just reuse the above `myPatternsCategory`?

Reply:
yeh, I didn't initially have that constant when I set that array item, and overlooked that more obvious change when switching to it, thanks.